WordPress/src/main/java/org/wordpress/android/models/Capability.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.models;
+
+/**
+ * Used to decide what can the current user do in a particular blog
+ * A list of capabilities can be found in: https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+ */
+public enum Capability {
+    EDIT_USERS("edit_users"), // Check if user can change another user's role
+    LIST_USERS("list_users"), // Check if user can visit People page
+    PROMOTE_USERS("promote_users"); // Check if user can invite another user
+
+    private final String label;
+
+    Capability(String label) {
+        this.label = label;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Hmm, seems to me that in that answer, the `If you want to print the enum constant in a user friendly way, you should use toString` is pretty compatible to my point. Notice the "user friendly way" part... In the `Capability` case, we're looking at the string to pass to the API instead, not what to put out on a debug log or even the UI.